.github/workflows/build-and-test.yml

```diff
@@ -125,3 +132,207 @@ jobs:
         with:
           name: collector-binaries
           path: ./bin
+  loadtestpre:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.splitloadtest.outputs.matrix }}
+    steps:
```

Comment:
`loadtestpre` does some pre-processing so we can simulate CircleCI testsharding on GitHub Actions. 
This job does the following: 
This job creates a json that is consumed by the `strategy.matrix` of `loadtest` 

1. Get a list of all loadtest tests using `make testbed-list-loadtest`
2. Split tests into groups of two (`TestIdleMode|TestBallastMemory`, `TestLog10kDPS|TestMetric10kDPS` etc)
3. Create a JSON with each test grouping. This JSON will be consumed by the [strategy.matrix](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) of the loadtest job using [`fromJSON()`](https://github.blog/changelog/2020-04-15-github-actions-new-workflow-features/). The strategy.matrix is how we get to run the loadtests in parallel